src/common/config/constants.ts

```diff
@@ -304,12 +305,22 @@ export const ExposedOperatorNames = [
 export type OperatorName = typeof OperatorNames[number];
 export type ExposedOperatorName = typeof ExposedOperatorNames[number];
 
-export const operatorLabels = (t: TFunction): { [key in ExposedOperatorName]: string } => ({
-  [OPERATOR_NAME_ODF]: t('ai:OpenShift Data Foundation'),
-  [OPERATOR_NAME_LVM]: t('ai:Logical Volume Manager'),
-  [OPERATOR_NAME_LVMS]: t('ai:Logical Volume Manager Storage'),
-  [OPERATOR_NAME_CNV]: t('ai:OpenShift Virtualization'),
-});
+export const operatorLabels = (
```

Comment:
There are still some old usages that are not passing the new parameters, please fix those too.

I checked and apparently `FailedOperatorsWarnings` is not being used anymore, so we can remove this file. HOwever, `ProgressBarAlerts` should be refactored.

Reply:
`FailedOperatorsWarnings` file removed. `ProgressBarAlerts` fixed. So now all usages of  the `operatorLabels` functions should be covered.